### ISSUE TITLE
Week of 9/20 Review Ticket

### ISSUE THREAD

**Issue by PrishaB** [PrishaB/Team-5-Blog#2]
Team 5
- Prisha
- Saumya
- Serafina
- Armaan

All our work is [here](https://prishab.github.io/Team-5-Blog/jupyter/2022/09/20/lesson.html)

**Comment by aaravarora01** [comment]
2.7+/3

Serafina - Created Quiz and Explained FRQ well with details. Showed clear understanding of this week's topics
Prisha - Added on well to FRQ and showed work with good explanation, Defined 2D Arrays
Armaan - In depth explanation of FRQ and all parts made sense, seems like he helped with quiz
Saumya - Explained Scrum Board and talked a bit about how the team collaborated

**Comment by jm1021** [comment]
Saumya - Refresher on If statements and loops.  Also spoke on methods and distinction from objects.  
Serafina - Discussed frog sim, introduction to problem and lots of content on screen. Part a.  for loop with conditions discussed.  Perhaps comments on code code of shortened explanation.  Part b.  discussed for loop with if and again long discussion for a few lines code.
Prisha - Discussed secret message problem.   Intro to problem.  Later Prisha came back and talked on 2D arrays.  Answered question on hacks and strong in technical language.   Discussing key part of code was a bit better.
Armaan - carried on with code overview of secret message.  Again I would suggest a couple of comments and less talking.  Pointing to code on screen is hard to follow.  Armaan spent time talking about players.  Nice adaptation, personalization.   Maybe too much on interest.  

Team built a scrum board.

Generally, good command of material shown.   Team has some flow and handoffs were good.   Just description were too long.   2.7 for everyone.

Still way over time!

**Comment by somaditya1** [comment]
Overall Group Grade: 2.7+
This grade is deserving because of the following attributes:
-Multiple FRQs have been implemented, signifying both key learning and effort
-Organized presentation tactics, each individual came prepared.
-Questions create interactive presentation that entertained and engaged the audience.
-In depth dissection of code, proving a thorough understanding of the FRQs.
-Scrum Board was also utilized to demonstrate crucial collaboration.
-Only problem was some of the members were quiet, making them hard to hear.

**Comment by Adi-K-Coding** [comment]
Cool that you included a quiz, frqs had a good thorough explanation, and the 2D array explanation also helped me understand the concept more in depth. Soccer player array was fun twist on monkeys. Good organization in roles and scrumboard. 2.7

**Comment by aaditgupta21** [comment]
- Good amount of FRQs that demonstrated their learning
- I like the refresher of Java in the beginning
- Quiz was a good addition to the presentation
- Explained each part of the Free Response Question Lesson
- I liked the lesson on arrays with the multiple examples
- Good use of Scrum Board and usage of the variety of roles

2.7/3

**Comment by TianbinLiu** [comment]
Nice presentation!
 
There are detailed descriptions for the different parts of code in the FRQ, 
Everyone's works are on the same page.
Everyone presented and explained his work.
All work is completed perfectly, good programming
had prepared for the presentation, for example slides

**Comment by boply** [comment]
- Nice concise presentation. 
- Good use of scrum to organize. 
- Quiz was a nice touch with some good examples.
- Good explaining of the FRQ. (I understood even though I didn't do that year :D)
